tests/src/Simple/Threading/Threading.cs

```diff
@@ -954,25 +951,46 @@ public static void RunJobsAfterThreadTimeout()
     }
 
     [Fact]
-    public static void ThreadLocalWorkQueueDepletionTest()
+    public static void WorkQueueDepletionTest()
     {
         ManualResetEvent e0 = new ManualResetEvent(false);
-        int count = 0;
-        int maxCount = Environment.ProcessorCount * 64;
+        int numLocalScheduled = 0;
+        int numGlobalScheduled = 0;
```

Comment:
Could initialize these two to 1 since one of each type is going to be scheduled below